MACHINE-LEARNING-SERVICE/src/controllers/objectRecognition.controller.js

```diff
@@ -0,0 +1,34 @@
+// Recognizes objects
+const recognizeObjects = (req, res) => {
+    const {files: images} = req;
+    let imagesploaded = images.map((image)=>{
+        return {
+            name: image.filename,
+            type: image.mimetype,
+            destination: (process.env.SERVER_URL + '/download-image/' + image.filename).replace(/ /g, '%20')
+        }
+    });
+    // TODO: calls the ObjectRecognition class
+
+    res.json({
+        msg: 'Images were uploaded.',
+        images: imagesploaded
+    });
+}
+
+// downloads image
```

Comment:
Omit the comment

Reply:
Please check again. I've already removed the comments.